contracts/gov/GovVote.sol

```diff
@@ -202,7 +202,11 @@ abstract contract GovVote is IGovVote, GovCreator {
                         false
                     );
 
-                    if (status == IGovValidators.ProposalState.Undefined) {
+                    if (
+                        status == IGovValidators.ProposalState.Undefined &&
+                        core.settings.validatorsVote &&
+                        govValidators.validatorsCount() > 0
```

Comment:
These 2 lines have to be in the `if (address(validators) != address(0))` and the latter if removed